### ISSUE TITLE
EA server connection error in BFV

### ISSUE THREAD

**Issue by agent-carl** [italomandara/CXPatcher#158]
"EA server connection error" I cant connect to the servers EA, and therefore can not enter the network game in Battlefield 5

>Crossover 23.6
>CXPatcher 4.7

**Comment by italomandara** [comment]
Does it work if it's unpatched?

**Comment by agent-carl** [comment]
yes

**Comment by italomandara** [comment]
Are you using d3dmetal or dxvk?

**Comment by agent-carl** [comment]
With dxvk very bad fps and lots of starters, so i use d3dmetal

**Comment by italomandara** [comment]
try downloading Crossover 23.7 beta if it does have the same issue, it could be msync

**Comment by agent-carl** [comment]
No, the clear beta version of the crossover does not have this issue
